guides/common/modules/con_puma-threads.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Puma_Threads_{context}"]
+= Puma Threads
+
+Less Puma threads lead to more memory usage for different scales on your {ProjectServer}.
+
+For example, we have compared these two setups:
+
+[width="100%",cols="50%,50%",options="header",]
+|===
+|{Project} VM with 8 CPUs, 40 GiB RAM |{Project} VM with 8 CPUs, 40 GiB RAM
+|`--foreman-foreman-service-puma-threads-min=0` |`--foreman-foreman-service-puma-threads-min=16`
+|`--foreman-foreman-service-puma-threads-max=16` |`--foreman-foreman-service-puma-threads-max=16`
+|`--foreman-foreman-service-puma-workers=2` |`--foreman-foreman-service-puma-workers=2`
+|===
+
+Setting the minimum Puma threads to `16` results in about 12% less memory as compared to `t_min=0`.
```

Comment:
Can you confirm @jhutar? Note that we are providing this only for Foreman 3.1+.

Reply:
Yep, we also recommend min threads == max threads.

We recommended 16 because we had a better results with it.

I agree tuning Puma workers is more straight forward.

This is what I released yesterday as an update to 6.10 tuning guide: https://github.com/RedHatSatellite/satellite-performance-tuning/commit/b87caa34b409aa36c440d9ef044c9fd8ce964ea0#diff-4d49ae92acb7629a016fdcecfa8cfa3458fdda0c93fd14cc1ebebe0cbfe703eeL298-R344

FYI Maximilian, please do not feel obligated to backport all these changes in 6.10 branch. I think we can do these once this huge PR is merged together.

Reply:
Thanks; I've added this!
I will backport this for Foreman 3.1, but obviously Satellite doesn't have to use it right away.